flocker/node/functional/test_docker.py

```diff
@@ -356,6 +398,7 @@ def image_built(image_name):
                 unit_name=unit_name,
                 image_name=image_name,
                 environment=Environment(variables=expected_variables),
+                force=True,
```

Comment:
> Also, what about other tests that create and then remove containers (ie, all the other tests using start_application)?
> 
> Couldn't they all also fail with this error?

Plausibly, yes - but the issue FLOC-1194 is only concerned with the environment variables test and is a "fix flaky test" category issue. So it possibly makes sense to not alter the behaviour of other tests using the Client change in this branch until such time as they are also identified as intermittently failing on the same device busy error.